support.go

```diff
@@ -5,6 +5,9 @@ import "encoding/json"
 type Support struct {
 	Files    []SupportFile    `yaml:"files,omitempty" json:"files,omitempty" validate:"dive"`
 	Commands []SupportCommand `yaml:"commands,omitempty" json:"commands,omitempty" validate:"dive"`
+	// Timeout is the seconds to wait for custom support commands to complete.
+	// If zero, the default timeout will be applied.
+	Timeout UintString `yaml:"timeout,omitempty" json:"timeout,omitempty"`
```

Comment:
ive been trying to add comments like this. i dont know if its valuable or not
https://github.com/replicatedhq/libyaml/blob/master/root_config.go#L42